<ui version="4.0">
 <class>AddEditCoffeeForm</class>
 <widget class="QDialog" name="AddEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить/Редактировать кофе</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="roastEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="groundCheck">
     <property name="text">
      <string>Молотый</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="descriptionEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="priceEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="volumeEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
